frontend/dockerfile/dockerfile_test.go

```diff
@@ -6852,19 +6887,27 @@ RUN echo foo >> /test
 }
 
 func testNamedImageContextScratch(t *testing.T, sb integration.Sandbox) {
-	integration.SkipOnPlatform(t, "windows")
 	ctx := sb.Context()
 
 	c, err := client.New(ctx, sb.Address())
 	require.NoError(t, err)
 	defer c.Close()
 
-	dockerfile := []byte(`
+	dockerfile := []byte(integration.UnixOrWindows(
+		`
 FROM busybox
 COPY <<EOF /out
 hello world!
 EOF
-`)
+`,
+		`	
+FROM nanoserver AS build
+FROM busybox
```

Comment:
why `FROM busybox again`?
BTW, for this one, can just do `fmt.Sprintf(`FROM %s ...`, UnixOrWindows("busybox", "nanoserver")`
since the difference is just the image.

Reply:
Good catch. This should be fixed now.